rental.rb

```diff
@@ -1,10 +1,16 @@
 class Rental
-  attr_accessor :date, :book, :person
+  attr_accessor :id, :date, :book, :person
 
   def initialize(date, book, person)
+    @id = Infor.generate_rental_id
     @date = date
     assign_book(book)
     assign_person(person)
+    Infor.rentals << self
+  end
```

Comment:
- [ ] I got an error when I tried to create a rental, kindly take a look at the image I have attached and make the necessary changes.

![rental](https://github.com/stephenkati/oop-school-lib/assets/53124548/0d3841f3-ee38-48dd-b57a-78583880e8bc)

Reply:
Oh I realised my problem with selecting user by id when creating a rental, thanks for the feedback @elisha2kyakpo1